Patch by requate@univention.de (Thu Sep 29 12:05:00 2009 +0200):

Fix for KOLAB/t_framework_HK_GW_Kolab__Server_RewriteExtend.diff

patch disabled, since bug was not reproducable (https://issues.kolab.org/issue3890)

--- a/horde-webmail/lib/Horde/Kolab/Server/Object/user.php	2009-10-03 20:37:03.000000000 +0200
+++ b/horde-webmail/lib/Horde/Kolab/Server/Object/user.php	2009-10-03 20:38:28.000000000 +0200
@@ -234,7 +234,7 @@
             }
             $server = $this->getServer('homeserver');
             if (is_a($server, 'PEAR_Error')) {
-                return $server;
+                $server = '';
             }
             if (empty($server)) {
                 $server = $_SERVER['SERVER_NAME'];
